### PR TITLE
django_1_5: remove

### PR DESCRIPTION
**Pull request by cko**
###### Motivation for this change
Django 1.5 is no longer developed and should therefor be removed (see: https://www.djangoproject.com/download/)

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
cc @lsix

**Comment by fpletz** [comment]
As  this version of Django has security vulnerabilities and is not maintained by upstream anymore it should also be removed from 16.09.

**Comment by lsix** [comment]
Hi (sorry for late reply).

At first sight, it looks good to me. Dropping django-1.5 would be nice. Maybe each individual updates can be separated into individual commits. I’ll review that PR properly ASAP (I have very limited access to my computer those days).

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -27726,14 +27703,15 @@ in {
...
+    version = "0.9.15";
 
     src = pkgs.fetchurl rec {
       url = "mirror://pypi/g/graphite-web/${name}.tar.gz";
-      sha256 = "472a4403fd5b5364939aee10e78f171b1489e5f6bfe6f150ed9cae8476410114";
+      sha256 = "1c0kclbv8shv9nvjx19wqm4asia58s3qmd9fapchc6y9fjpjax6q";
     };
 
-    propagatedBuildInputs = with self; [ django_1_5 django_tagging whisper pycairo ldap memcached ];
+    propagatedBuildInputs = with self; [ django_tagging whisper pycairo ldap memcached ];
```
Comment (lsix) [inline pkgs/top-level/python-packages.nix]: Even if `django_tagging` has `django` as a propagated build input, `django` should be listed as direct build dependency of graphite_web.

**Review by lsix — CHANGES_REQUESTED** [review]
Hi, thanks for the PR.

Could it be possible to:
- explicitly keep `django` as a [propagated] build input of `graphite_web`,
- separate the work in one commit for each affected derivation?

With that, it will be OK for me.

**Comment by cko** [comment]
[x] done

**Pull request merged**